.travis.yml

```diff
@@ -14,7 +14,6 @@ matrix:
 branches:
     only:
         - master
-        - 36lts
```

Comment:
this reminds me we should probably remove the `36lts` check from Avocado-vt's `.travis.yml` (it's used to check py2.6 branch).

Reply:
True!